time_tracker_api/project_types/project_types_model.py

```diff
@@ -33,3 +35,15 @@ def __init__(self):
             SQLCRUDDao.__init__(self, ProjectTypeSQLModel)
 
     return ProjectTypeSQLDao()
+
+
+container_definition = {
+    'id': 'project_type',
+    'partition_key': PartitionKey(path='/customer_id'),
+    'unique_key_policy': {
+        'uniqueKeys': [
```

Comment:
I think we need to include parent_id and maybe also tenant_id.

Reply:
The tenant id is the parent partition_key, so it kind of defines a domain for those unique keys (it is kind of unique by default). The `parent_id` cannot be unique, because this is a tree structure: every branch has multiple branches or leaves not only one.